GitUI/CommandsDialogs/FormFileHistory.cs

```diff
@@ -284,7 +290,7 @@ private void LoadFileHistory()
 
                 if (AppSettings.FullHistoryInFileHistory)
                 {
-                    res.revision = string.Concat(" --full-history --simplify-merges ", res.revision);
+                    res.revision = string.Concat(" --full-history ", AppSettings.SimplifyMergesInFileHistory ? "--simplify-merges " : string.Empty, res.revision);
```

Comment:
https://github.com/gitextensions/gitextensions/wiki/Guidelines%3A-Coding-Patterns

Reply:
For this PR, this can pass. This function requires a major refactoring to use GitArgumentBuilder.

Reply:
Agree with Gerhard. Let's refactor it separately